pulpanalytics/views.py

```diff
@@ -104,6 +104,8 @@ def _accumulator(prev, value):
         return value | {"count": prev["count"] + value["count"]}
 
     context["demography"] = []
+    if daily_summary is None:
```

Comment:
What about moving this to the outer function and saving he qs and then checking it roughly [here](https://github.com/pulp/analytics.pulpproject.org/blob/98a914c62565ea9fe0b008a152d15d1fdf3f1372/pulpanalytics/views.py#L192). I think that would be more explicit and better.

Reply:
Depends on what you want to accomplish. This function in its current state secures that the variable "demography" is made available to the template and then check if it can actually be filled with data.

Reply:
Also where it lives today is in line with the bailout two lines down for another way of "no data available".

Reply:
I believe it's cleaner that way.